app/assets/stylesheets/home.scss

```diff
@@ -20,18 +20,31 @@ body.home.index {
     }
 
     .leaflet-popup {
-      position: relative;
       background: white;
       padding: 1em;
       z-index: -100 !important;
-      margin-top: -4.75em;
+      margin-top: -(rem(104/2));
+      margin-left: 2em;
+
+      &:before {
+        content: "";
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets